include/flox/core/util.hh

```diff
@@ -23,6 +24,17 @@
 #include <nlohmann/json.hpp>
 
 
+/* -------------------------------------------------------------------------- */
+
+/* Backported from C++20b. */
+
+/** @brief Helper type for `std::visit( overloaded { ... } );` */
+template<class... Ts>
```

Comment:
I don't understand what this does, and I'm not sure the comment really helps.

Reply:
It's used by a later PR.

The same is true for `hasPrefix` and `isUInt`.

`std::visit( overloaded { ... } );` is the C++ equivalent of a type matcher.
For it to work in C++2a ( our version ) this snippet is required.

Reply:
Added more inline docs.